tutorials/desktop/install-jre/install-jre.md

```diff
@@ -0,0 +1,97 @@
+---
+id: install-jre
+summary: Installation of Java Runtime Environment (JRE) on Ubuntu
+categories: desktop
+tags: tutorial,installation,ubuntu,desktop,java,jre
+difficulty: 1
+status: Published
+published: 2017-12-02
+author: Aden Padilla <adenpadilla@gmail.com>
+
+---
+
+# Install Java Runtime Environment (JRE)
+
+## Overview
+Duration: 1:00
+
+Java Runtime Environment (JRE) is required to run Java programs. There are several versions of Java and the ones we can install on our Ubuntu machines are OpenJRE and Oracle JRE. Both of them are similar and should not have any functional difference but some prefer OpenJRE over Oracle JRE as it is easier to install because it requires no additional repositories to be added. However, there are cases where you might need the other one.
```

Comment:
Suggestion for this paragraph: 

"The Java Runtime Environment (JRE) is required to run Java programs. While there are multiple JRE variants available, the two most popular ones on Ubuntu are OpenJRE and Oracle JRE. Using one over the other should not create any functional difference in most applications, however, some prefer OpenJRE over Oracle JRE as it doesn't contain closed-source components and is maintained as part of the Ubuntu archive, with easier installation and upgrades."